ownership-chain/runtime/src/precompiles/mod.rs

```diff
@@ -39,7 +39,7 @@ where
 			hash(8),
 			hash(9),
 			hash(1027),
-			hash(1028),
```

Comment:
this needs to be documented somewhere so that we don't use this address for other precompile